ingestion/src/main/java/feast/store/serving/redis/FeatureRowToRedisMutationDoFn.java

```diff
@@ -42,17 +43,24 @@ public FeatureRowToRedisMutationDoFn(Map<String, FeatureSet> featureSets) {
 
   private RedisKey getKey(FeatureRow featureRow) {
     FeatureSet featureSet = featureSets.get(featureRow.getFeatureSet());
-    Set<String> entityNames =
+    List<String> entityNames =
         featureSet.getSpec().getEntitiesList().stream()
             .map(EntitySpec::getName)
-            .collect(Collectors.toSet());
+            .sorted()
```

Comment:
With this `sorted` we make an assumption that `entities` field in `RedisKey` is sorted alphabetically according to the entity name? We need to update the Feast Serving code too right, to follow the same asssumption?

Reply:
Oh and do you think it's better to have it sorted by entity name or ensure that the order of entity follow that defined in FeatureSetSpec?

Reply:
I had this conversation with Willem before in the previous PR, I think because we consider 2 featuresets with different order of features to be the same always sorting the key by alphabetical order is more deterministic